ledger-service/http-json/src/main/scala/com/digitalasset/http/domain.scala

```diff
@@ -131,6 +133,29 @@ object domain extends com.daml.fetchcontracts.domain.Aliases {
 
   final case class PartyDetails(identifier: Party, displayName: Option[String], isLocal: Boolean)
 
+  final case class UserRights(canActAs: List[Party], canReadAs: List[Party], isAdmin: Boolean)
+
+  object UserRights {
+    def fromListUserRights(input: Vector[UserRight]): UserRights = {
+      val (canActAs: Vector[String], remaining1) = input.partitionMap {
```

Comment:
[This is not a type annotation](https://discuss.daml.com/t/v-ty-in-a-pattern-is-not-an-annotation-scala-ritual/1450); please remove every form like this.

Reply:
Thank you very much for the insight!

Reply:
done